cross/expat/Makefile

```
@@ -1,9 +1,10 @@
 PKG_NAME = expat
 PKG_VERS = 2.2.7
-PKG_EXT = tar.bz2
-PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://downloads.sourceforge.net/project/$(PKG_NAME)/$(PKG_NAME)/$(PKG_VERS)
-PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_EXT = tar.gz
+PKG_DIST_NAME = R_2_2_7.$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/lib$(PKG_NAME)/lib$(PKG_NAME)/archive/refs/tags
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = lib$(PKG_NAME)-R_2_2_7/$(PKG_NAME)
```

With the `tar.bz2` source of github you get exactly the same file as on sourceforge.
Then you can omit `expat_pre_configure` and revert the digests file.

```suggestion
PKG_DIST_SITE = https://github.com/libexpat/lib$(PKG_NAME)/releases/download/R_$(subst .,_,$(PKG_VERS))
```

---

There are a lot of github repos that provide there custom release packages (that e.g. do not need configure to generate the Makefile). The *.zip and *.tar.gz source packages are autogenerated by github and are often the wrong choice to build the packages.

---

Thanks, Done.

---

We about to hit 1k forks on this repo! :partying_face: